### ISSUE TITLE
Error :Not Implement nn.SpatialUpSamplingBilinear

### ISSUE THREAD

**Issue by XunShape** [clcarwin/convert_torch_to_pytorch#28]
hi, I wanna convert vrn-unguided.t7 in https://github.com/AaronJackson/vrn 
and it said :Not Implement nn.SpatialUpSamplingBilinear
actually，i got vrn_unguided.pth and vrn_unguided.py
but a part of vrn_unguided.py was wrong,
like this:
LambdaReduce(lambda x,y: x+y), # CAddTable,
	LambdaMap(lambda x: x, # ConcatTable,
		nn.Sequential( # Sequential,
			nn.Sequential( # Sequential,
				nn.Conv2d(256,256,(1, 1)),
				nn.BatchNorm2d(256),
				nn.ReLU(),
			),
			nn.Conv2d(256,200,(1, 1)),
			#,
			 ,
			n,
			n,
			.,
			S,
			p,
			a,
			t,
			i,
			a,
			l,
			U,
			p,
			S,
			a,
			m,
			p,
			l,
			i,
			n,
			g,
			B,
			i,
			l,
			i,
			n,
			e,
			a,
			r,
			 ,
			N,
			o,
			t,
			 ,
			I,
			m,
			p,
			l,
			e,
			m,
			e,
			n,
			t,
			,,
			
,

**Comment by XunShape** [comment]
[vrn_unguided_py.zip](https://github.com/clcarwin/convert_torch_to_pytorch/files/1628700/vrn_unguided_py.zip)

**Comment by crazySyaoran** [comment]
Have the same problem, also the vrn code.
Have you solved?

**Comment by alexandru-dinu** [comment]
You have to replace 
```
else:
    s += '# ' + name + ' Not Implemented,\n'
```
with
```
else:
    s += ['# ' + name + ' Not Implemented,\n']
```

Notice the brackets. That's because of `map(lambda x: '\t{}'.format(x), s)`; that function must be mapped onto each _sublist_ from s.

Also, [this line](https://github.com/clcarwin/convert_torch_to_pytorch/blob/master/convert_torch.py#L253) must be changed in order to avoiding adding a stray `,`.

One very quick fix is:

```
s = map(lambda x: '{},\n'.format(x) if not x.strip()[0] == '#' else '{}'.format(x), s)
```

**Comment by Goulustis** [comment]
could try to go to the generated .py file and replace it with nn.Upsample
